homeassistant/components/image_processing/facebox.py

```diff
@@ -74,18 +97,18 @@ def process_image(self, image):
         try:
             response = requests.post(
                 self._url,
-                json=encode_image(image),
+                json={"base64": encode_image(image)},
                 timeout=9
```

Comment:
Make the 9 a constant defined at the begging of the file

Reply:
ok